charts/dremio-module/Chart.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v2
+name: dremio-module-multi-chart
+description: A Helm chart for installing a module for Fybrik in Kubernetes
```

Comment:
A Helm chart for installing a **Dremio** module for Fybrik in Kubernetes

Reply:
Fixed